frontend/src/components/Secrets/GSecretDialogDDNS.vue

```diff
@@ -0,0 +1,245 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+ -->
+
+<template>
+  <g-secret-dialog
+    v-model="visible"
+    :data="secretData"
+    :secret-validations="v$"
+    :secret="secret"
+    vendor="netlify-dns"
+    create-title="Add new DDNS (RFC2136) Secret"
+    replace-title="Replace DDNS (RFC2136) Secret"
+  >
+    <template #secret-slot>
+      <div>
+        <v-text-field
+          v-model="server"
+          color="primary"
+          label="<host>:<port> of the authorive DNS server"
+          :error-messages="getErrorMessages(v$.server)"
+          type="text"
+          variant="underlined"
+          @update:model-value="v$.server.$touch()"
+          @blur="v$.server.$touch()"
+        />
+      </div>
+      <div>
+        <v-text-field
+          v-model="tsigKeyName"
+          color="primary"
+          label="TSIG Key Name"
+          :error-messages="getErrorMessages(v$.tsigKeyName)"
+          type="text"
+          variant="underlined"
+          @update:model-value="v$.tsigKeyName.$touch()"
+          @blur="v$.tsigKeyName.$touch()"
+        />
+      </div>
+      <div>
+        <v-text-field
+          v-model="tsigSecret"
+          color="primary"
+          label="TSIG Secret"
+          :error-messages="getErrorMessages(v$.tsigSecret)"
+          :append-icon="hideTSIGSecret ? 'mdi-eye' : 'mdi-eye-off'"
+          :type="hideTSIGSecret ? 'password' : 'text'"
+          variant="underlined"
+          @click:append="() => (hideTSIGSecret = !hideTSIGSecret)"
+          @update:model-value="v$.tsigSecret.$touch()"
+          @blur="v$.tsigSecret.$touch()"
+        />
+      </div>
+      <div>
+        <v-text-field
+          v-model="zone"
+          color="primary"
+          label="Zone"
+          :error-messages="getErrorMessages(v$.zone)"
+          type="text"
+          variant="underlined"
+          @update:model-value="v$.zone.$touch()"
+          @blur="v$.zone.$touch()"
+        />
+      </div>
+      <div>
+        <v-select
+          v-model="tsigSecretAlgorithm"
+          color="primary"
+          label="TSIG Secret Algorithm"
+          :items="secretAlgorithmItems"
+          variant="underlined"
+        />
+      </div>
+    </template>
+
+    <template #help-slot>
```

Comment:
help text should not cover more than 50% of the screen

Reply:
Added max-width
Also added a gradient to visualize that there is more content that can be scrolled into view